Mastermindzh/react-cookie-consent
# Overlay develop feature

This adds a (default 30% opaque) page overlay.

To test default behavior:
Create a `CookieConsent` component and add it to your project.
Add the `overlay` property to the component.
Observe in the project that there is now a page-spanning overlay that is 30% opaque by default.
```javascript
      <CookieConsent
        location="bottom"
        cookieName="myAwesomeCookieName2"
        expires={999}
        overlay
      >
        This website uses cookies to enhance the user experience.
      </CookieConsent>
    </div>
```

You can also add `overlayClasses` and `overlayStyle` properties the same as other properties that use similar features.
